Test/regressions/UInt.sol

```diff
@@ -0,0 +1,44 @@
+pragma solidity >=0.4.24 <0.6.0;
+
+
+// Test uint implementation
+contract UintTest {
+  uint x;
+  uint256 x256;
```

Comment:
Can we add a couple of variables of type uint[], mapping (int => uint) and possibly uint[][] to see if we handle nested uints well?

Reply:
And possibly when passing a uint as a call argument?

Reply:
Done. I manually checked that all assertion actually failed when changing the conditions on variables. Should we include test with assertion failures?